src/table/components/virtualized-table/utils/get-cell-style.ts

```diff
@@ -2,10 +2,13 @@ import { SelectionStates, SELECTION_STYLING, StylingDefaults } from '../../../co
 import { BodyStyle } from '../types';
 
 const getCellStyle = (
-  isHoveringOnRow: boolean,
+  isRowHovered: boolean,
+  showHoverEffect: boolean,
   cellSelectionState: SelectionStates,
   bodyStyle: BodyStyle
 ): BodyStyle => {
+  const isHoveringOnRow = isRowHovered && showHoverEffect;
```

Comment:
naming should probably be somethign like
```suggestion
  const showHoverEffetxt = isRowHovered && isHoverEnabled;
```